inkscape/src/main/assembly/dist/lace_ground.inx

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <inkscape-extension xmlns="http://www.inkscape.org/namespace/inkscape/extension">
 
-	<_name>Lace Ground</_name>
+	<_name>Ground from Template</_name>

```

Comment:
I have called my generated ground descriptions “templates” on my website since they were first posted which is why I chose the name “Template”.  

I don’t have a quick answer on how to distinguish between the two approaches.  I will poke around some tools that I use and see how they describe the different user interactions.

The main difference I think is that one tool renders a diagram for pairs (i.e., one line represents one pair) and the other tool renders a diagram for individual threads (i.e. one line represents one thread).  They both load a template for the repeated pattern but the output is different.

From: jo-pol 
Sent: Monday, April 13, 2015 3:28 AM
To: jo-pol/DiBL 
Cc: Veronika Irvine 
Subject: Re: [DiBL] contributor role figured out (#4)

In inkscape/src/main/assembly/dist/lace_ground.inx:

> @@ -1,7 +1,7 @@
>  <?xml version="1.0" encoding="UTF-8"?>
>  <inkscape-extension xmlns="http://www.inkscape.org/namespace/inkscape/extension">
>  
> -	<_name>Lace Ground</_name>
> +	<_name>Ground from Template</_name>
Might be confusing with my templates. I suppose you mean your generated matrices. While my templates are InkScape files. How do we disambiguate?

—
Reply to this email directly or view it on GitHub.